osquery/filesystem/CMakeLists.txt

```diff
@@ -12,8 +12,16 @@ elseif(LINUX)
   )
 endif()
 
+if(WIN32)
+  set(PROCESS_FAMILY "windows")
```

Comment:
If you need to set this in `./osquery/core/CMakeLists.txt`, do it in the top-level `CMakeLists.txt`.